content/blog/2025-02-28-tmil-13/index.md

```diff
@@ -0,0 +1,245 @@
++++
+title = "Linebender in January 2025"
+authors = ["Kaur Kuut", "Daniel McNab", "Tom Churchman", "Raph Levien"]
++++
+
+Linebender is an informal open-source organization working on various projects to advance the state of the art in GUI for [the Rust programming language](https://rust-lang.org).
+
+## Xilem
+
+Xilem is our flagship GUI project, inspired by SwiftUI.
+It lets you build user interfaces declaratively by composing lightweight views together, and will diff them to provide minimal updates to a retained layer.
+
+Xilem saw decent direct progress in January and of course Xilem also benefits from all the progress in the rest of our stack which is detailed in later sections.
+
+- [xilem#420][], [xilem#833][]: Added emoji picker example.
+- [xilem#806][]: Made `Pod` more flexible to allow creating a wider variety of wrapper widgets.
+- [xilem#828][]: Moved transformation related boilerplate to a single `View`.
```

Comment:
I think transforms were also added in january: https://github.com/linebender/xilem/pull/753

So maybe
```suggestion
- [xilem#753][]: Support transforms for each widget.
- [xilem#828][]: Moved transformation related boilerplate to a single `View`.
```

Reply:
Yeah it was indeed under Masonry. That said, it did touch a significant part of Xilem as well so it doesn't hurt to mention it twice. I added it under Xilem as well. 👍